demos/interactive_face_detection_demo_gapi/interactive_face_detection_gapi.hpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+///////////////////////////////////////////////////////////////////////////////////////////////////
+
+#include <gflags/gflags.h>
+
+static const char help_message[] = "Print a usage message";
+static const char input_video_message[] = "Required. Path to a video file (specify \"cam\" to work with camera).";
+static const char output_video_message[] = "Optional. Path to an output video file.";
+static const char face_detection_model_message[] = "Required. Path to an .xml file with a trained Face Detection model.";
+static const char age_gender_model_message[] = "Optional. Path to an .xml file with a trained Age/Gender Recognition model.";
+static const char head_pose_model_message[] = "Optional. Path to an .xml file with a trained Head Pose Estimation model.";
+static const char emotions_model_message[] = "Optional. Path to an .xml file with a trained Emotions Recognition model.";
+static const char facial_landmarks_model_message[] = "Optional. Path to an .xml file with a trained Facial Landmarks Estimation model.";
+static const char plugin_message[] = "Plugin name. For example, CPU. If this parameter is specified, " \
+"the demo will look for this plugin only.";
+static const char target_device_message[] = "Optional. Target device for Face Detection network (the list of available devices is shown below). " \
+"Default value is CPU. The demo will look for a suitable plugin for a specified device.";
+static const char target_device_message_ag[] = "Optional. Target device for Age/Gender Recognition network (the list of available devices is shown below). " \
+"Default value is CPU. The demo will look for a suitable plugin for a specified device.";
+static const char target_device_message_hp[] = "Optional. Target device for Head Pose Estimation network (the list of available devices is shown below). " \
+"Default value is CPU. The demo will look for a suitable plugin for a specified device.";
+static const char target_device_message_em[] = "Optional. Target device for Emotions Recognition network (the list of available devices is shown below). " \
+"Default value is CPU. The demo will look for a suitable plugin for a specified device.";
+static const char target_device_message_lm[] = "Optional. Target device for Facial Landmarks Estimation network " \
+"(the list of available devices is shown below). Default value is CPU. The demo will look for a suitable plugin for device specified.";
+// TODO: Support this option
```

Comment:
If an option is not supported, don't add it in the first place.

Reply:
Removed all references to unsupported features
Put one TODO instead